overlord/servicestate/quota_control.go

```diff
@@ -58,7 +67,59 @@ func EnsureQuotaUsability() (restore func()) {
 	}
 }
 
+func checkMemoryCGroupEnabled() {
+	memoryCGroupError = memoryCGroupEnabled()
+}
+
+// added to enable path update for testing purposes
```

Comment:
Are you sure that this comment is correct? It seems to me that if this function is not called the `checkMemoryCGroupEnabled()` function won't be called either, so `memoryCGroupError` will never be set; and I believe it's something we want to have always, not only in tests.

Probably, however, we want to keep the comment as is, and instead change the code to always call `checkMemoryCGroupEnabled()`; but I'm not sure from where.

Reply:
This seems to be a merge error. Added the call to init() as in the previous iteration.